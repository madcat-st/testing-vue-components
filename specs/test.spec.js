import { mount } from '@vue/test-utils'
import TestComponent from '@/test'

test('Mount a vue component', () => {
// const wrapper = mount(TestComponent)
const wrapper = mount(TestComponent, {
  propsData: {
    value: 'Something new'
  }
})

//console.log(wrapper)
expect(wrapper.html()).toMatchSnapshot()
})
