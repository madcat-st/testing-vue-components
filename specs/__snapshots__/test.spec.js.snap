// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mount a vue component 1`] = `
<div>
  <h1>Hello Something new</h1>
</div>
`;
